modules/ztp-generating-install-and-config-crs-manually.adoc

```diff
@@ -64,6 +64,13 @@ $ mkdir -p ./site-install
 ----
 include::snippets/ztp_example-sno.yaml[]
 ----
++
+[NOTE]
```

Comment:
Ideally this information would be included as a callout. However as part of Reference Design Specification development,  snippets are now included as YAML files imported directly. Adding callouts to the yaml files would increase complexity and potentially cause issues with reuse of the snippets.